<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>721</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView_tableDB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>211</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_search">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UpdateDB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Update DB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_re_login_singup">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>re-LogIn-SingUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setting">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_edit_data">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save edit data</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>961</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
